app/org/sagebionetworks/bridge/dao/FPHSExternalIdentifierDao.java

```diff
@@ -0,0 +1,22 @@
+package org.sagebionetworks.bridge.dao;
+
+import java.util.List;
+
+import org.sagebionetworks.bridge.models.accounts.ExternalIdentifier;
+import org.sagebionetworks.bridge.models.accounts.FPHSExternalIdentifier;
+
+public interface FPHSExternalIdentifierDao {
+
+    public boolean verifyExternalId(ExternalIdentifier externalId);
+    
+    public void registerExternalId(ExternalIdentifier externalId);
+    
+    public void unregisterExternalId(ExternalIdentifier externalId);
+    
+    public List<FPHSExternalIdentifier> getExternalIds();
+    
+    public void addExternalIds(List<FPHSExternalIdentifier> externalIds);
+    
+    public void deleteAll();
```

Comment:
While this is less likely to come up, I'd strongly prefer a deleteById() call instead of a deleteAll() call. This way, if someone wants to test the external ID stuff locally, they can do that without interfering with the unit tests and vice versa.